ReText/dialogs.py

```diff
@@ -79,3 +79,28 @@ def handleTextChanged(self, value):
 			button.setEnabled(False)
 		else:
 			button.setEnabled(True)
+
+
+class urlPopup(QLabel):
+
+	def __init__(self, parent):
+		super().__init__(parent)
+		self.p = parent
+
+		self.setStyleSheet('''
+			border: 1px solid #64323232;
+			border-radius: 3px;
+			background: #FAFAFAFA;
+		''')
+		self.fh = self.p.fontMetrics().height()
+		self.setVisible(False)
+
+	def popup(self, url):
+		if url:
+			self.setText(url)
+			bot = self.p.rect().bottom()
+			tw = self.p.fontMetrics().horizontalAdvance(url)
```

Comment:
Please use longer names for the variables to make them more meaningful: `self.fontHeight`, `bottom`, `textWidth`.